app/helpers/batch_tools.rb

```diff
@@ -0,0 +1,63 @@
+module BatchTools
+
+	def save_or_get_errors(post_manipulations)
```

Comment:
`save_or_get_errors` is too generic of a helper method name. What would a more specific name be?